testsuite/features/trad_baremetal_discovery.feature

```diff
@@ -48,7 +48,7 @@ Feature: Bare metal discovery
     And I should see a "Number of disks" text
```

Comment:
Bump copyright date to 2019 ;-)

Reply:
? I didn't touched that file.
Is our of the scope of that PR.